public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme

```diff
@@ -72,6 +72,29 @@ function hdbt_subtheme_preprocess_organization_information_block(array &$variabl
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Adds number of open vacancies for job listing title field template.
+ *
+ * @throws \Drupal\Core\TypedData\Exception\MissingDataException
```

Comment:
Does it really?

Reply:
`$node->get('field_jobs')?->first()` throws it. See `\Drupal\Core\TypedData\ListInterface.php`.

Reply:
Right, carry on!